demo-plugin/src/main/kotlin/org/jetbrains/research/refactoringDemoPlugin/util/PsiUtil.kt

```diff
@@ -33,3 +38,35 @@ fun concatFiltered(
     array2.filterTo(filteredList, condition)
     return filteredList.toTypedArray()
 }
+
+/*
+    Extracts all Java classes from the project.
+ */
+fun extractClasses(project: Project?): MutableList<PsiClass> {
```

Comment:
Should `project` really be nullable? It's not nullable in the only usage in this branch.